packages/core/src/components/tag-input/_tag-input.scss

```diff
@@ -17,34 +21,39 @@ $ti-icon-padding-large: ($pt-input-height-large - $pt-icon-size-large) / 2;
   height: auto;
   min-height: $pt-input-height;
   padding-right: 0;
-  padding-left: $ti-padding;
+  padding-left: $tag-input-padding;
 
   .#{$ns}-tag-input-icon {
     // margins to center icon in one-line input
-    margin-top: $ti-icon-padding;
-    margin-right: $ti-icon-padding;
-    margin-left: $ti-icon-padding - $ti-padding;
+    margin-top: $tag-input-icon-padding;
+    margin-right: $tag-input-icon-padding;
+    margin-left: $tag-input-icon-padding - $tag-input-padding;
     color: $pt-icon-color;
   }
 
   .#{$ns}-tag-input-values {
-    @include pt-flex-container(row, $ti-padding);
+    @include pt-flex-container(row, $tag-input-padding);
     flex-wrap: wrap;
     align-items: center;
     // fill vertical height
     align-self: stretch;
-    margin-top: $ti-padding;
-    margin-right: $ti-icon-padding;
+    margin-top: $tag-input-padding;
+    margin-right: $tag-input-icon-padding;
 
     > * {
-      margin-bottom: $ti-padding;
+      margin-bottom: $tag-input-padding;
     }
   }
 
   .#{$ns}-tag {
+    position: relative;
     // NOTE: in order to wrap long words, you must set explicit width on TagInput,
     // or use .#{$ns}-fill CSS class modifier.
     overflow-wrap: break-word;
+
+    &.#{$ns}-active {
+      @include focus-outline(0);
```

Comment:
i kinda think we want to adjust outline offset on _all_ tags, not just in tag inputs:

![image](https://user-images.githubusercontent.com/464822/39782395-e88985aa-52c6-11e8-9c22-d86605116d55.png)
(note how the outline touches the neighboring tags)

![image](https://user-images.githubusercontent.com/464822/39782415-f6c0c28c-52c6-11e8-8ee6-3555adf66bbb.png)

so this change should move back to core tag styles.

Reply:
I don't think that's the right move, but did it anyway, Not sure what to do with outline design in general